pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -193,10 +193,17 @@ func buildLogsTimeSeriesFilterQuery(fs *v3.FilterSet, groupBy []v3.AttributeKey,
 				case v3.FilterOperatorContains, v3.FilterOperatorNotContains:
 					columnName := getClickhouseColumnName(item.Key)
 					val := utils.QuoteEscapedString(fmt.Sprintf("%v", item.Value))
-					conditions = append(conditions, fmt.Sprintf("%s %s '%%%s%%'", columnName, logsOp, val))
+					conditions = append(conditions, fmt.Sprintf("lower(%s) %s lower('%%%s%%')", columnName, logsOp, val))
```

Comment:
Does the same `lower` mentioned for body work for regular attributes also?

Reply:
There are two points to this
* Previously we were utilising `ilike` for attributes as well. The index was not at all getting used.
* Now it will not work as well since we are using lower(attribute_name) but we will have to address this by making our string indexes to lower.


but here I am finding an issue that if we make it lowercase than the '=' ,  'in' should also be chaned to lowercase which might cause issue with 'Appname' and 'appname'

Reply:
Do you want to address this independently?

Reply:
Yeah can do that, will remove the changes for attributes then for now.

Reply:
have made the changes, now it will only apply for body

Reply:
https://github.com/SigNoz/signoz/blob/dfc7f827d8d3917d0d9b62a6809f3c4e58018ac2/pkg/query-service/app/logs/v3/json_filter.go#L76